packages/os/os.spec

```diff
@@ -25,6 +25,7 @@ Source6: metricdog-toml
 Source7: host-ctr-toml
 Source8: oci-default-hooks-json
 Source9: cfsignal-toml
+Source10: autoscaling-warm-pool-k8s-toml
```

Comment:
This doesn't really have anything to do with Kubernetes, so the config file shouldn't have `k8s` in the name.